src/index.html

```diff
@@ -1,14 +1,235 @@
+          </div>
+          <div class="menue__item">
+            <img src="images/menue__items__img/95b745f6ce6a45903c3ed254afd72e46-w550-276.png" alt="">
+            <p class="menue__item__restaurant">WOKWEI</p>
+            <ul class="menue__item__tag one__tag">
+              <li>
+                 Китайська
+              </li>
+              <li>
+                 Азіатська
+              </li>
+            </ul>
+            <p class="menue__item__time"> 40-50 Min </p>
+          </div>
+          <div class="menue__item">
+            <img src="images/menue__items__img/95b745f6ce6a45903c3ed254afd72e46-w550-277.png" alt="">
+            <p class="menue__item__restaurant">Milk Bar</p>
+            <ul class="menue__item__tag double__tag">
+              <li>
+                 Десерти
+              </li>
+              <li>
+                 Європейська
+              </li>
+            </ul>
+            <p class="menue__item__time"> 25-35 Min </p>
+          </div>
+          <div class="menue__item">
+            <img src="images/menue__items__img/95b745f6ce6a45903c3ed254afd72e46-w550-2778.png" alt="">
+            <p class="menue__item__restaurant">Mimosa Brookly Pizza</p>
+            <ul class="menue__item__tag double__tag">
+              <li>
+                 Італійська
+              </li>
+              <li>
+                 Піца
+              </li>
+              <li>
+                 Американська
+              </li>
+            </ul>
+            <p class="menue__item__time"> 15-25 Min </p>
+          </div>
+          <div class="menue__item">
+            <img src="images/menue__items__img/95b745f6ce6a45903c3ed254afd72e46-w550-278.png" alt="">
+            <p class="menue__item__restaurant">To Be</p>
+            <ul class="menue__item__tag double__tag">
+              <li>
+                 Японська
+              </li>
+              <li>
+                 Азіатська
+              </li>
+              <li>
+                 Суші
+              </li>
+            </ul>
+            <p class="menue__item__time"> 25-35 Min </p>
+          </div>
+          <div class="menue__item">
+            <img src="images/menue__items__img/95b745f6ce6a45903c3ed254afd72e46-w550-279.png" alt="">
+            <p class="menue__item__restaurant">Druzi Cafe (вул. Прорізна)</p>
+            <ul class="menue__item__tag double__tag">
+              <li>
+                 Бургери
+              </li>
+            </ul>
+            <p class="menue__item__time"> 30-40 Min </p>
+          </div>
+
+        </div>
+      </div>
+    </div>
+  </main>
+  <footer>
+    <div class="box-content">
+      <div class="footer__logo">
+        <a href="#" class="logo">
+          uber <span>eats</span>
+        </a>
+      </div>
+      <div class="footer__info">
+        <ul>
+            <li class="select__lang">
+              <img class="globe" src="images/icon/Shape1.svg" alt="">
+              <select class="" name="">
```

Comment:
1. make `<select>` dropdown width the same as in the mockup
2. https://prnt.sc/otel0l this arrow has to work when you click on it